spec/std/http/server/handlers/static_file_handler_spec.cr

```diff
@@ -177,6 +177,183 @@ describe HTTP::StaticFileHandler do
     end
   end
 
+  context "when a Range header is provided" do
+    describe "valid Range header" do
+      it "serves a byte range" do
+        headers = HTTP::Headers{"Range" => "bytes=0-2"}
+        response = handle HTTP::Request.new("GET", "/range.txt", headers)
+
+        response.status_code.should eq(206)
```

Comment:
These specs should also test that the `Content-Range` header has the correct value for single-part requests, or is [absent for multi-part ones](https://www.rfc-editor.org/rfc/rfc9110.html#section-15.3.7.2-1)